### PR TITLE
Add Technical Team to Governance pg from #112

### PR DESCRIPTION
**Pull request by lunacodes**

### CODE REVIEW

File: content/governance.md [hunk 1]
```diff
@@ -21,3 +21,32 @@ The CWL leadership team consists of the following people, listed in alphabetical
...
+* Michael R. Crusoe, ELIXIR-NL
+  * <https://github.com/mr-c>
+  * <https://matrix.to/#/@mr-c:matrix.org>
+  * <https://orcid.org/0000-0002-2961-9670>
+* Bogdan Gavrilović, Seven Bridges
+  * <https://github.com/bogdang989>
+  * <https://matrix.to/#/@bogdan_gavrilovic:matrix.org>
+  * <https://orcid.org/0000-0003-1550-1716>
+  * <https://github.com/orgs/common-workflow-language/teams/tech_team>
+  * <https://matrix.to/#/#cwltool-dev:matrix.org>
```
Comment (mr-c) [inline content/governance.md:45]: These links should go in the explanation paragraph on line 27

Comment (lunacodes) [reply content/governance.md:45]: I just added the GitHub link to the paragraph on line 27 (https://github.com/common-workflow-language/cwl-website/pull/117/commits/34f2667da3ad0687a34d42243ac3d941f4640822) , but wasn't sure how to work in the Matrix link. Probably better if someone else writes that in.

**Comment by mr-c** [comment]
@lunacodes I added icons for the profile link; maybe there is a cleaner way to do that using CSS?

**Comment by lunacodes** [comment]
> @lunacodes I added icons for the profile link; maybe there is a cleaner way to do that using CSS?

@mr-c sure - I can style them up a bit. Is there a reason that only the Technical Team has orcid icons? I think they'd look good in the leadership section as well. 

Also, I think the section needs/would benefit from a "CWL Technical Team" heading - so I'll add that in, if that's ok.

**Comment by lunacodes** [comment]
These should be good to go! I increased the icon sizes a bit, and gave them some spacing, as well as gave the icons a line break for mobile and tablet screens (it came out very hectic and cluttered otherwise). Also tidied up the markdown a good deal!

**Mobile:**

governance-mobile

**Tablet:**

governance-tablet

**Desktop:**

governance-desktop

**Review by mr-c — APPROVED** [review]

**Comment by mr-c** [comment]
Thank you very much @lunacodes !

**Pull request merged**